stub.sh

```diff
@@ -366,7 +389,9 @@ __stub_call() {
 
   local index="$(__stub_index "$cmd")"
   if [ -n "$index" ]; then
-    eval "STUB_${index}_CALLS+=(\"\$args\")"
+    # Send call to the main process
```

Comment:
if we are in the main process this shouldn't be really necessary I've decided to always do it because I thought this would be more consistent.